src/test/java/org/opentripplanner/middleware/utils/ItineraryUtilsTest.java

```diff
@@ -201,6 +211,76 @@ public void canAddIgnoreRealtimeParam() throws URISyntaxException {
         }
     }
 
+    /**
+     * Check whether certain itineraries match.
+     */
+    @ParameterizedTest
+    @MethodSource("createItineraryMatchingTestCases")
+    public void testItineraryMatches(ItineraryMatchTestCase testCase) {
+        Assertions.assertEquals(
+            testCase.shouldMatch,
+            ItineraryUtils.itinerariesMatch(testCase.previousItinerary, testCase.newItinerary),
+            testCase.name
+        );
+    }
+
+    private static List<ItineraryMatchTestCase> createItineraryMatchingTestCases() throws CloneNotSupportedException {
```

Comment:
I kind of want this method name to be `createItineraryComparisonTestCases`. The word Matching makes me think they should all represent examples of matching itins.

Reply:
Fixed in ffc3b49b3ad576d936b04758d572efdf3f217ab2